src/main/java/com/minecolonies/rankup/Plugininfo.java

```diff
@@ -5,11 +5,11 @@
  */
 public class Plugininfo
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this class "final" or add a public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2974)